### DISCUSSION TITLE
Οργανίσμος SDY60-2023

### DISCUSSION THREAD

**Discussion by gkalogeitonas** [sdy60/syllabus#95] [Show and tell]
Έκανα ένα οργανισμό  για το Μέρος Α https://github.com/SDY60-2023
και ένα Repository για το webring https://github.com/SDY60-2023/webring.

Βάλτε από κάτω τα email σας να σας καλέσω  και όποιος επιθυμεί να αναλάβει  και  διαχειριστή να με ενημερώσει.

**Comment by IoannisTheodoropoulos** [comment]
Ευχαριστούμε! Eιμαι ο std159317@ac.eap.gr

**Comment by std159312** [comment]
Καλησπέρα είμαι ο std159312@ac.eap.gr

**Comment by Konssven** [comment]
Καλησπέρα και απο εμένα είμαι ο std162798@ac.eap.gr

**Comment by std133524** [comment]
Επίσης ευχαριστούμε το email μου είναι το std133524@ac.eap.gr

**Comment by Maurphal** [comment]
Καλησπέρα! Κάνω αυτό το σχόλιο ως αίτημα ενσωμάτωσης στον οργανισμό. 
Επισης Γιώργο,  αν θες κανε με και εμενα admin, να το ψαχνω να το μαθω σιγα σιγα.. ;) 
Ευχαριστώ

**Comment by nikos7887** [comment]
Καλημέρα Γιώργο, καλή εβδομάδα. Σε ευχαριστούμε.

**Comment by ghost** [comment]
Ευχαριστούμε.(Αίτημα ενσωμάτωσης ) std159316@ac.eap.gr
